package.json

```diff
@@ -4,31 +4,31 @@
 	"scripts": {
 		"preinstall": "npx only-allow yarn",
 		"postinstall": "husky install",
-		"clean:": "yarn workspaces foreach -pv run clean",
-		"build:": "yarn workspaces foreach -pvt run build",
-		"start:": "yarn workspaces foreach -piv run start",
-		"deploy:": "yarn workspaces foreach -piv run deploy",
-		"lint:": "essex lint --fix",
+		"clean": "yarn workspaces foreach -pv run clean",
+		"build": "yarn workspaces foreach -pvt run build",
+		"start": "yarn workspaces foreach -piv run start",
+		"deploy": "yarn workspaces foreach -piv run deploy",
+		"lint": "essex lint --fix",
 		"git_is_clean": "essex git-is-clean",
 		"prettify:": "essex prettify",
-		"test:": "NODE_OPTIONS=--experimental-vm-modules yarn jest",
-		"ci": "run-s build: lint: test: git_is_clean",
+		"jest": "NODE_OPTIONS=--experimental-vm-modules yarn jest",
```

Comment:
seems like we should still have a root-level `test`, even if it is an alias for `jest` - this would still avoid recursion, right?

Reply:
Because there are no package-specific tests, making "test' just run "jest" is fine